pkg/reconciler/v1alpha1/revision/resources/queue.go

```diff
@@ -131,6 +131,9 @@ func makeQueueContainer(rev *v1alpha1.Revision, loggingConfig *logging.Config, a
 		}, {
 			Name:  "USER_PORT",
 			Value: strconv.Itoa(int(userPort)),
+		}, {
+			Name:  "AUTOSCALER_NAMESPACE",
+			Value: "knative-serving",
```

Comment:
The queue sidecar's container spec created by the function `makeQueueContainer`. I can add an environment value to transmit the `knative-serving`. But the namespace at here is also hardcode.  I don't know is this right? @mattmoor